src-tauri/src/app_config.rs

```diff
@@ -629,6 +635,11 @@ impl AppConfig {
         self.airdrop_tokens.clone()
     }
 
+    pub fn swaps_enabled(&self) -> Option<bool> {
+        warn!(target: LOG_TARGET, "Accessed Old config | swaps_enabled");
```

Comment:
we're not meant to add stuff here anymore i don't think :( 
just adding it in config_core should be a-ok